draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1274,7 +1274,7 @@ Fixed-Length Dedicated Counter Seeding:
   counter rollovers.
 
 Fixed-Length Dedicated Counter Length:
-: Care MUST be taken to select a counter bit-length that can properly handle
+: Take care to select a counter bit-length that can properly handle
```

Comment:
While I like the notion that careless implementations are incorrect 😄, I'm not sure that saying people MUST take here works here.  Probably better to just take it out.